solc/CommandLineInterface.cpp

```diff
@@ -658,19 +660,39 @@ void CommandLineInterface::handleCombinedJSON()
 			ostringstream unused;
 			contractData["asm"] = m_compiler->streamAssembly(unused, contractName, m_sourceCodes, true);
 		}
+		if (requests.count("srcmap"))
+		{
+			auto map = m_compiler->sourceMapping(contractName);
+			contractData["srcmap"] = map ? *map : "";
+		}
+		if (requests.count("srcmap-runtime"))
+		{
+			auto map = m_compiler->runtimeSourceMapping(contractName);
+			contractData["srcmap"] = map ? *map : "";
```

Comment:
I didn't saw that sorry.
seems that it should be
contractData["srcmap-runtime"] = map ? *map : "";